streams/src/main/java/org/apache/kafka/streams/processor/internals/StoreChangelogReader.java

```diff
@@ -368,18 +368,6 @@ private boolean allChangelogsCompleted() {
             .allMatch(metadata -> metadata.changelogState == ChangelogState.COMPLETED);
     }
 
-    // for stream thread to decide to transit to normal processing
-    // TODO K9113: called by the task manager to decide when to transit to standby updating.
-    boolean allActiveChangelogsCompleted() {
```

Comment:
After reading your code I think this function returning a boolean is not needed so I removed it.